### DISCUSSION TITLE
Thank you.

### DISCUSSION THREAD

**Discussion by geekm0nkey** [mathoudebine/turing-smart-screen-python#292] [General]
@mathoudebine Thank you. I really didn't like the original program that came with this little screen and your python code offered much more than just an alternate method to display content. It gave me the insight that I was really missing out when it comes to coding python. I have been coding various languages for the past 25+ years, never once gave python a second look till now. So once again Thanks for the work you put into this. (and any others that contributed.)

**Comment by mathoudebine** [comment]
Thanks for your message! This project started as a "why not" without any real purpose other than satisfy my own curiosity about what could be done with Turing smart screen. This was the early age of smart screens 2 years ago, and the first working code stayed for 8 months on this repo without any interest until smart screens where sold everywhere. Then many people began creating PR to add feature and create a complete program (system-monitor) from the basic original code, and it even became the first result when searching "turing smart screen" on google 🚀 sometimes I can't even find enough time to read all new discussions and PR, but I'm amazed by the creativity of the community!
I'm happy you enjoyed tweaking it, and Python is really a powerful and popular language for this type of cross-platform tools. As a software developer it has become my favorite programming/scripting language for personal projects. Have fun!

**Comment by LoCoCrUzN** [comment]
Totally agree with geekm0nkey. Great work. I could never get the Manufactures Software to keep running for more than a couples of hours, It would start giving me errors. I would haft to delete it and redownload it and almost every day. I only have one request to be about to choose between Celsius or Fahrenheit.
